hack/ocm-minikube-ramen.sh

```diff
@@ -144,13 +154,10 @@ exit_stack_push unset -v ocm_ramen_samples_git_ref
 application_sample_namespace_and_s3_deploy()
 {
 	kubectl create namespace busybox-sample --dry-run=client -o yaml | kubectl --context ${1} apply -f -
-	kubectl --context $1 -n busybox-sample apply -f https://raw.githubusercontent.com/$ocm_ramen_samples_git_path/ocm-ramen-samples/$ocm_ramen_samples_git_ref/subscriptions/busybox/s3secret.yaml
 }
 exit_stack_push unset -f application_sample_namespace_and_s3_deploy
 application_sample_namespace_and_s3_undeploy()
 {
-	kubectl --context $1 -n busybox-sample delete -f https://raw.githubusercontent.com/$ocm_ramen_samples_git_path/ocm-ramen-samples/$ocm_ramen_samples_git_ref/subscriptions/busybox/s3secret.yaml
-	date
 	kubectl --context ${1} delete namespace busybox-sample
 	date
 }
```

Comment:
(nit) I think these routines and the calls to them can be removed now since the namespace was created early for the s3secret.  Instead the application namespace can be included in the application deployment like it was prior to https://github.com/RamenDR/ocm-ramen-samples/commit/a2015092c37cb3c4164096354e7ccb5ed2404acb

Reply:
Ack! This also led to #240